### PR TITLE
Fix some small nits I found in BlinkNG

### PR DESCRIPTION
**Pull request by chrishtr**
- Refer consistently to BlinkNG
- A few typo fixes

### CODE REVIEW

**Comment by chrishtr** [comment]
@rachelandrew @szager-chromium

**Comment by chrishtr** [comment]
friendly ping

File: site/en/blog/blinkng/index.md [hunk 1]
```diff
@@ -175,10 +175,10 @@ To understand why this is important, and where else it may lead, we need to cons
...
 
 Waiting in the wings is [Off-main-thread Compositing](https://docs.google.com/document/d/1BfIqMcJvmMFdk8e_KKeOEofz16610Mql6gmfeFhrsZk/preview), with the goal of making the rendering engine match the illustration by moving _layerization_ off the main thread, and onto a worker thread. Like Non-Blocking Commit, this will reduce congestion on the main thread by diminishing its rendering workload. A project like this would never have been possible without the architectural improvements of Composite After Paint.
 
-And there are more projects in the pipeline (pun intended)! We finally have a foundation that makes it possible to experiment with redistributing rendering work, and we are very excited to see what's possible!
\ No newline at end of file
+And there are more projects in the pipeline (pun intended)! We finally have a foundation that makes it possible to experiment with redistributing rendering work, and we are very excited to see what's possible!
```
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:184]: Is this an end-of-file newline diff? I think this was previously a problem.

Comment (szager-chromium) [reply site/en/blog/blinkng/index.md:184]: Everything else lgtm.

Comment (una) [reply site/en/blog/blinkng/index.md:184]: Looks fine to me in preview, is this a concern @rachelandrew?

**Review by una — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/blinkng/index.md (hunk 1 above)
Comment (rachelandrew) [reply site/en/blog/blinkng/index.md:184]: I think it's fine, if the preview is building it should be.

**Pull request merged**